packages/core/services/google-maps-types.ts

```diff
@@ -634,3 +634,65 @@ export interface MapRestriction {
   latLngBounds: LatLngBounds | LatLngBoundsLiteral;
   strictBounds?: boolean;
 }
+
+export interface Geocoder {
+  geocode: (request: GeocoderRequest, googleCallback: (results: GeocoderResult, status: GeocoderStatus) => void) => void;
+}
+
+export interface GeocoderRequest {
```

Comment:
please alphabetize the properties

Reply:
Sorry, just to clarify: alphabetize the interface in sorting order? Sorry for the dumb question

Reply:
Alright, done.